components/postmark/actions/send-email-with-template/send-email-with-template.mjs

```diff
@@ -0,0 +1,51 @@
+import postmark from "../../postmark.app.mjs";
+
+let objSharedProps = {};
+postmark.methods.listSharedProps().forEach((propName) => {
+  objSharedProps[propName] = {
+    propDefinition: [
+      postmark,
+      propName,
+    ],
+  };
+});
+
+export default {
+  key: "postmark-send-email-with-template",
+  name: "Send an email with template",
+  description: "Send an email with Postmark using a template",
+  version: "0.1.23",
```

Comment:
Should this version be `0.0.1`?

```suggestion
  version: "0.0.1",
```

Reply:
Adjusted version numbers to 0.0.1 for the new action and two new sources.